### ISSUE TITLE
Gracefully handle certain exceptions

### ISSUE THREAD

**Issue by somesh-kumar** [eclipse/vorto#2062]
There are quite a few exceptions in production logs due to:

- Session invalid
- User does not have permissions to access a model
- Invalid characters used in search

**Actual**: ERROR is thrown

**Expected**: Handle these exceptions gracefully with an INFO message. Identify if there are more such exceptions which do not need to be thrown as an ERROR.

**Comment by mena-bosch** [comment]
Having a look.

**Comment by mena-bosch** [comment]
Hi @somesh-kumar @aedelmann , I've started having a look and so far, I can categorize a couple of those cases, based on the error messages I got from AWS.
However, I'm having a hard time replicating anything consistently - maybe either of you can help me do that locally in an ad-hoc "discovery" session.

- Session invalid --> suspect AbstractRepositoryController#unAuthorized --> change logging level would do. 

- User does not have permissions to access a model --> maybe this matches one or more of the exception handlers in AbstractRepositoryOperation#doInSession? In which case, either lower the logging level to warn or see caller if rethrown?

- Invalid characters in search --> we had an initial conversation with Somesh but I forgot where we left it. It seems that spamming a few search requests containing the % character in the prod server consistently fails, but no e-mail seems to be sent after the threshold. The same fails locally, but is intercepted at Spring level. URL-encoding the text at front-end controller level doesn't seem to do anything. @somesh-kumar can we discuss replication again, and the last actions you've taken after our internal discussion? Your last statement was: "still waiting for an exception email to come to me, hoping we get the same error"

**Comment by mena-bosch** [comment]
PR [here](https://github.com/eclipse/vorto/pull/2147). 
As mentioned, my change set is weaksauce for now: it only lowers some logging levels to warning. 
Per se, this should not be dangerous given the use cases, but it might contribute to clear some of the AWS threshold occurrences. 
My take is we can iterate further once this is merged, and see if there is something missed out.

**Comment by aedelmann** [comment]
Hi @mena-bosch 
With your fix, lets keep an eye on the logs and see if your changes improved it. Nevertheless, I would recommend to specify an Exception Handling concept, with all possible errors that could occur in the system and define the measures that need to be taken. Lets create a separate issue for that. I would close this issue.
